Content.Shared/Movement/Systems/SharedJetpackSystem.cs

```diff
@@ -117,21 +116,22 @@ private void OnJetpackToggle(EntityUid uid, JetpackComponent component, ToggleJe
         if (args.Handled)
             return;
 
-        if (TryComp<TransformComponent>(uid, out var xform) && !CanEnableOnGrid(xform.GridUid))
+        if (!CanEnable(Transform(uid).Coordinates))
         {
             if (_timing.IsFirstTimePredicted)
-                _popups.PopupEntity(Loc.GetString("jetpack-no-station"), uid, args.Performer);
+                _popups.PopupEntity(Loc.GetString("jetpack-under-gravity"), uid, args.Performer);
 
             return;
         }
 
         SetEnabled(component, !IsEnabled(uid));
     }
 
-    private bool CanEnableOnGrid(EntityUid? gridUid)
+    private bool CanEnable(EntityCoordinates location)
     {
-        return gridUid == null ||
-               (!HasComp<GravityComponent>(gridUid));
+        var tile = location.GetTileRef();
+        return tile == null || tile.Value.IsSpace() ||
+               !HasComp<GravityComponent>(location.GetGridUid(EntityManager));
```

Comment:
Don't use tile helper methods and don't use the tileref helper method without at least passing in the tiledefmanager.